ifrs17-template/PracticalUseCases/ActualsOutsideThePeriod/ActualsUseCaseReports.ipynb

```diff
@@ -0,0 +1,1093 @@
+                "\n"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Data Import"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "We call the ActualsUseCaseDataImport notebook, where the data is imported."
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "#!import \"ActualsUseCaseDataImport\""
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Best Estimate Present Value of Cashflows",
+                "\n",
+                "\n"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "We start by looking at the Best Estimate Present Value of Cashflows of MZ 2.1 Written (base case) for the 3 reporting periods."
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## 2020 12"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var pv = ifrs17.PresentValues;",
+                "\npv.ReportingNode = \"CH\";",
+                "\npv.ReportingPeriod = (2020, 12);",
+                "\npv.CurrencyType = CurrencyType.Contractual;",
+                "\npv.ColumnSlices = new string[]{\"GroupOfContract\", \"AmountType\"};",
+                "\npv.DataFilter = new []{ (\"GroupOfContract\", \"MZ2.1\") };",
+                "\n(await pv.ToReportAsync) with {Height = 720}"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## 2021 6"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "var pv = ifrs17.PresentValues;",
```

Comment:
You can use the previously defined pv variable and just update the reporting period. 
When we'll have the app mode we can think of a way to refactor it.